lib/curly/presenter.rb

```diff
@@ -52,6 +52,10 @@ def initialize(context, options = {})
           end
         end
 
+        if value.respond_to?(:call)
+          value = value.call
```

Comment:
Nah, I like it being on the presenter instance.

How about

```ruby
if default_values.key?(name) && value.respond_to?(:call)
  ...
end
```

Alternatively, we could make the feature use blocks, e.g.

```ruby
presents(:author) { @comment.author }
```